assets/src/edit-story/app/history/useHistoryReducer.js

```diff
@@ -53,7 +54,12 @@ const reducer = ( size ) => ( state, { type, payload } ) => {
 			};
 
 		default:
-			throw new Error( `Unknown history reducer action: ${ type }` );
+			const text = sprintf(
+				/* translators: %s: Type of error. */
+				__( 'Unknown history reducer action: %s', 'amp' ),
```

Comment:
This is a tricky one. I'd argue that these kind of errors are more developer facing and should not be translatable. I could not even properly translate this anyway.

Reply:
Do you want me to remove this?

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/4143/commits/d4cd0a55ce8b81f0ad639c1f2c7a244444fafc34